kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/type/ScaleOffsetType.java

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Eurotech and/or its affiliates and others
+ * 
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ * 
+ * Contributors:
+ *  Eurotech
+ ******************************************************************************/
+package org.eclipse.kura.type;
```

Comment:
Is it better to put this into the `org.eclipse.kura.channel` package?

Reply:
The new enum seems only related to Asset channel configuration and scale and offset to me and not general like DataType and TypedValue. All of the scale and offset related APIs are in the `org.eclipse.kura.channel` package.